decidim-proposals/app/views/decidim/proposals/admin/proposals/_form.html.erb

```diff
@@ -9,7 +9,7 @@
     </div>
 
     <div class="row column hashtags__container">
-      <%= form.text_area :body, rows: 10, class: "js-hashtags", hashtaggable: true, value: form_presenter.body(extras: false).strip %>
+      <%= form.text_area :body, rows: 10, class: "js-hashtags js-mentions", hashtaggable: true, value: form_presenter.body(extras: false).strip %>
```

Comment:
I think that mentions can not be done in the body of a proposal... do you really need to add this css class?

Reply:
removed